### PR TITLE
chore: Lock postgres to 11.12

### PR DESCRIPTION
**Pull request by gagantrivedi**
Standardize PostgreSQL version to 11.12

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ test-api:
     image: python:3.7-slim
     stage: test
     services:
-        - postgres:10.9-alpine
```
Comment (gagantrivedi) [inline .gitlab-ci.yml:16]: Any reason why this was locked to 10.9(apart from the fact that it was the latest version at the time)?

Comment (dabeeeenster) [reply .gitlab-ci.yml:16]: We can probably remove the .gitlab-ci file now - what do you think @matthewelwell ?

Comment (matthewelwell) [reply .gitlab-ci.yml:16]: Yep, might as well remove (but probably do it in a separate PR to keep the git history clean)

Comment (dabeeeenster) [reply .gitlab-ci.yml:16]: #227

**Pull request merged**